packages/client/src/components/button/Button.styled.tsx

```diff
@@ -2,16 +2,28 @@ import styled from 'styled-components';
 
 import { color, font, space } from '../../variables';
 
-export const ButtonBaseStyles = styled.button`
+import { ButtonProps } from './Button';
+
+export const ButtonBaseStylesWrapper = styled.div`
+  display: grid;
+`;
+
+export const ButtonBaseStyles = styled.button<ButtonProps>`
   background: transparent;
   color: ${color.dodgerblue};
+  ${(props) => (props.color === 'dodgerBlue' ? `color: ${color.dodgerblue}` : `color: ${color.darkred}`)};
   cursor: pointer;
   font-size: ${font.size.base};
   font-weight: ${font.weight.bold};
   padding: ${space.x3} ${space.x6};
   border: 2px solid ${color.dodgerblue};
+  ${(props) =>
+    props.border === 'dodgerBlue' ? `border: 2px solid ${color.dodgerblue}` : `border: 2px solid ${color.darkred}`};
   border-radius: 5px;
   transition: 0.25s ease-in-out;
+  ${(props) => props.align === 'start' && `justify-self: start;`}
+  ${(props) => props.align === 'center' && `justify-self: center;`}
+  ${(props) => props.align === 'end' && `justify-self: end;`}
```

Comment:
this will work but here we have 3 times calling functions for the same reason, pls just do it in one call.